scripts/extract_all_raw.py

```diff
@@ -0,0 +1,57 @@
+import os
```

Comment:
Could you also add a shebang such as the following?

```
#!/usr/bin/env bash
```

Reply:
Done.